webroot/css/cake.css

```diff
@@ -350,20 +350,33 @@ pre {
 }
 
 .top-bar.expanded .title-area {
-    background: #01545b;
+    background: #ba232a;
+}
+
+.top-bar.expanded .title-area:before {
```

Comment:
`.title-area:before` is fine, too. 
Just out of curiosity, is less specific best practice?